konrad/radiation/arts.py

```diff
@@ -294,10 +256,10 @@ def calc_spectral_irradiance_field(self, atmosphere, t_surface):
         self.ws.spectral_irradiance_fieldFromSpectralRadianceField()
 
         return (
-            self.ws.f_grid.value.copy(),
-            self.ws.p_grid.value.copy(),
-            self.ws.spectral_irradiance_field.value.copy(),
-            self.ws.trans_field.value[:, 1:, 0].copy().prod(axis=1),
+            self.ws.f_grid.value.value.copy(),
+            self.ws.p_grid.value.value.copy(),
+            self.ws.spectral_irradiance_field.value.value.copy(),
```

Comment:
The ARTS types such as `Vector` don't have a copy method. They need to be converted into a numpy array first. Two options to do that:
```
self.ws.f_grid.value.value[:].copy()
```
or
```
np.copy(self.ws.f_grid.value.value)
```

Reply:
Sorry, I didn't notice the second `.value` when reviewing the code. That is a possibility as well. But maybe `[:]` is nicer to read.